src/main/java/com/palantir/stash/stashbot/urlbuilder/StashbotUrlBuilder.java

```diff
@@ -103,4 +108,16 @@ private String mask(String str) {
             return str;
         }
     }
+    
+    public String getJenkinsJobUrl(Repository repo, JobTemplate jt) throws SQLException {
+        RepositoryConfiguration rc = cps.getRepositoryConfigurationForRepository(repo);
+        JenkinsServerConfiguration jsc = cps.getJenkinsServerConfiguration(rc.getJenkinsServerName());
+        // XXX this is broken with folders, I think
```

Comment:
Oh yeah, you already reviewed the fix here: https://github.com/terabyte/stashbot/pull/15